whitepaper/whitepaper.css

```diff
@@ -1,7 +1,7 @@
 @import url(https://fonts.googleapis.com/css?family=Roboto:100);
```

Comment:
@deanpress I noticed there's also a `whitepaper.scss` and `whitepaper.css.map`, but I'm not sure how these are managed.  Do I need to worry about them?

Reply:
You can delete those in this PR